src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,58 @@
+import { FC, useEffect, useState } from 'react';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { Loader } from './Loader';
+import { PeopleTable } from './PeopleTable';
+
+export const PeoplePage: FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [hasError, setHasError] = useState('');
```

Comment:
Better to name it `errorMessage` because your store `string` here, not a `boolean` value